mne/report/report.py

```diff
@@ -354,10 +354,14 @@ def _fig_to_img(fig, *, image_format='png', own_figure=True):
 
     # https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html
     kwargs = dict()
+    pil_kwargs = dict()
     if image_format == 'webp':
-        kwargs['pil_kwargs'] = dict(lossless=True, method=6)
+        pil_kwargs.update(lossless=True, method=6)
     elif image_format == 'png':
-        kwargs['pil_kwargs'] = dict(optimize=True, compress_level=9)
+        pil_kwargs.update(optimize=True, compress_level=9)
+    if pil_kwargs:
+        # matplotlib modifies the passed dict, which is a bug
```

Comment:
Opened https://github.com/matplotlib/matplotlib/pull/24657 but in practice we won't ever really need to remove this line since it'll be very quick